bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/discovery/ThingDiscoveryService.java

```diff
@@ -109,10 +113,15 @@ private void addLight(String lightID, LightMessage light) {
             return;
         }
 
-        if (light.uniqueid.isEmpty()) {
-            logger.warn("No unique id reported for light {} ({})", light.modelid, light.name);
-            return;
-        }
+        Map<String, Object> properties = new HashMap<>();
+        properties.put(Thing.PROPERTY_FIRMWARE_VERSION, light.swversion);
+        properties.put(Thing.PROPERTY_VENDOR, light.manufacturername);
+        properties.put(Thing.PROPERTY_MODEL_ID, light.modelid);
+
+        if (light.ctmax != null)
```

Comment:
The standard openhab coding format requires that all if statements should have brackets.